jupyterlab/gst/ui-tests/README.md

```diff
@@ -0,0 +1,117 @@
+# Test
+
+The test will produce a video to help debugging and check what happened.
+
+To execute integration tests, you have two options:
+
+- use docker-compose (cons: needs to know and use docker) - this is a more reliable solution.
+- run tests locally (cons: will interact with your JupyterLab user settings)
+
+## Test on docker
+
+1. Compile the extension:
+
+```
+jlpm install
+jlpm run build:prod
+```
+
+2. Execute the docker stack in the example folder:
+
+```
+docker-compose -f ../end-to-end-tests/docker-compose.yml --env-file ./ui-tests/.env build
+docker-compose -f ../end-to-end-tests/docker-compose.yml --env-file ./ui-tests/.env run --rm e2e
+docker-compose -f ../end-to-end-tests/docker-compose.yml --env-file ./ui-tests/.env down
+```
+
+## Test locally
+
+1. Compile the extension:
+
+```
+jlpm install
+jlpm run build:prod
+```
+
+2. Start JupyterLab _with the extension installed_ without any token or password
+
+```
+jupyter lab --ServerApp.token= --ServerApp.password=
+```
+
+3. Execute in another console the [Playwright](https://playwright.dev/docs/intro) tests:
+
+```
+cd ui-tests
+jlpm install
+npx playwright install
+npx playwright test
```

Comment:
<img width="758" alt="Screen Shot 2021-09-29 at 00 03 58" src="https://user-images.githubusercontent.com/25267873/135177238-d64da24c-ff26-4733-852d-a693ae1190c2.png">

I got an error on the npx playwright install, but when testing it still worked :)

Reply:
Are you using macOS 10.13?